usagereportsv4/usage_reports_v4.go

```diff
@@ -3346,12 +3346,18 @@ func UnmarshalSnapshotListFirst(m map[string]json.RawMessage, result interface{}
 // SnapshotListNext : Reference to the next page of the search query if any.
 type SnapshotListNext struct {
 	Href *string `json:"href,omitempty"`
+
+	Offset *string `json:"offset,omitempty"`
```

Comment:
@jdatechong is this change the result of just manually adding this field to the struct, or did you re-generate the SDK code and this was literally the only change to the file?
If the former, please re-generate from the API definition.

Reply:
@padamstx when I regenerated the SDK in an earlier commit, it removed the offset field from SnapshotListNext; however, I'm being told that the offset field was used somewhere in https://github.com/IBM-Cloud/terraform-provider-ibm and it breaks its build.

Reply:
Why was it removed by the previous re-gen?  I suspect it's because that schema property was removed from the API definition.  If that's true, then it needs to be added back to the API definition and then a re-gen performed using the updated API definition.
I really do not want manual changes in generated SDK code to start polluting the SDK project. It is likely to hurt us in the future.  The API definition is our source of truth for the API.

Reply:
The production version of the API definition (https://github.ibm.com/cloud-api-docs/billing/blob/production/metering-reporting.v4.yaml) doesn't contain an "offset" field in the SnapshotList schema.

Reply:
Oh, I was re-generating using the [master branch which doesn't seem to have an offset field in the next field](https://github.ibm.com/cloud-api-docs/billing/blob/9de2b96b53fd78407d4c02b25cc596fb9631247b/metering-reporting.v4.yaml#L4064-L4070). Re-generating using the production branch adds in the offset field that I manually added in above.

Reply:
I think the code within the SDK projects should be in sync with the production version of the API definition, right?